app/src/main/java/org/p2p/wallet/user/interactor/UserInteractor.kt

```diff
@@ -74,11 +76,12 @@ class UserInteractor(
         }
     }
 
-    suspend fun loadUserTokensAndUpdateLocal(fetchPrices: Boolean) {
+    suspend fun loadUserTokensAndUpdateLocal(fetchPrices: Boolean): List<Token.Active> {
         val newTokens = userRepository.loadUserTokens(tokenKeyProvider.publicKey, fetchPrices)
         val cachedTokens = mainLocalRepository.getUserTokens()
 
         updateLocalTokens(cachedTokens, newTokens)
+        return getUserTokens()
```

Comment:
I am not sure if it's a good idea. 
Imho it's out of this method responsibility.
I would make a separate call in presenter instead of here

Reply:
okay, will fix it

Reply:
okay, will fix it